tarodb/asset_minting.go

```diff
@@ -535,6 +573,36 @@ func fetchAssetSprouts(ctx context.Context, q PendingAssetStore,
 	return taroCommitment, nil
 }
 
+// fetchAssetMetas attempts to fetch the asset meta reveal for each of the
+// passed assets.
+func fetchAssetMetas(ctx context.Context, db PendingAssetStore,
+	assets []*asset.Asset) (map[asset.SerializedKey]*proof.MetaReveal, error) { // nolint:lll
```

Comment:
nit: break before `error` return type instead of adding nolint directive?

Reply:
Looks weird visually imo, but done.

Reply:
Could now also use the new `AssetMetas` type to shorten the function definition.